src/Containers-OrderPreservingDictionary/CTOrderPreservingIdentityStringDictionary.class.st

```diff
@@ -1,5 +1,20 @@
 "
-An IdentityDictionary version that uses == instead of = for key comparing.
+I am an IdentityDictionary version that uses == instead of = for String key comparing. That means I consider two keys to be equal only if they are the same object. 
+
+I also returns empty strings instead of raising errors when absent keys/values are accessed. See CTOrderPreservingStringDictionary comment for example.
```

Comment:
What's the difference between this one and a normal `CTOrderPreservingIdentityDictionary`?
Why should I use this one and not the other (and vice-versa)?

I'm not saying this class is bad, but the guy reading the documentation will ask herself that question ;)

Reply:
Totally agree.